azurerm/resource_arm_virtual_machine.go

```diff
@@ -356,6 +357,19 @@ func resourceArmVirtualMachine() *schema.Resource {
 					},
 				},
 			},
+			"additional_capabilities": {
```

Comment:
could we [add this new block to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/virtual_machine.html.markdown#L95)

Reply:
added documentation for the above block